dmcaf/dm_runner.py

```diff
@@ -103,47 +109,36 @@ def run_experiment(self, experiment_id: str, model_configs: List[Dict[str, Any]]
             guidance_scale = config.get('guidance_scale', 7.5)
             num_inference_steps = config.get('num_inference_steps', 50)
             generator = torch.Generator(device="cpu").manual_seed(config.get('seed', 42))
-
-            print(f"Loading model: {model_name}")
-
-            pipeline_kwargs = {}
+            pipeline_kwargs = {} # Additional kwargs for pipeline initialization
             if vae_name:
                 print(f"Loading VAE: {vae_name}")
                 pipeline_kwargs['vae'] = AutoencoderKL.from_pretrained(vae_name)
 
-            if "stable-diffusion-3" in model_name:
```

Comment:
Why have we removed the stable diffusion 3 handler? StableDiffusion 3 models can't be run without their dedicated pipeline object